stable/redis/ci/redis-lib-values.yaml

```diff
@@ -4,7 +4,7 @@
 image:
   registry: docker.io
   repository: redis
-  tag: '4.0.11'
+  tag: '5.0.5'
```

Comment:
Please undo this change

Reply:
Flag is not supported on this version
See https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/helm_charts/15100/pull-charts-e2e/1144314462543548416/build-log.txt
```
docker://redis-08ug45oq09:Container failed liveness probe.. Container will be killed and recreated.
I0627 19:08:34.757]   Warning  Unhealthy               8m53s (x9 over 9m38s)  kubelet, gke-jenkins-default-pool-51cafeba-f5cd  Liveness probe failed: 
I0627 19:08:34.757] Warning: Using a password with '-a' option on the command line interface may not be safe.
I0627 19:08:34.757] Unrecognized option or bad number of args for: '--no-auth-warning'
```

Reply:
hmm.. You're right! Skip this comment!